ampersand-form-view.js

```diff
@@ -24,13 +27,12 @@ module.exports = View.extend({
     initialize: function(opts) {
         opts = opts || {};
         this.el = opts.el;
-        this.validCallback = opts.validCallback || this.validCallback || noop;
-        this.submitCallback = opts.submitCallback || this.submitCallback || noop;
+        this.validCallback = opts.validCallback || this.validCallback || null;
```

Comment:
Those `null` values are redundant, as it'll fallback to `undefined` anyway if user doesn't pass `opts` or it won't exist already.